Makefile

```diff
@@ -44,9 +44,12 @@ ios:
 	@echo "Done building."
 	@echo "Import \"$(GOBIN)/Kusd.framework\" to use the library."
 
-test: all
+test: all lint
```

Comment:
I think that test should not include lint - different areas.
ex: you have a local branch that needs to be tested several times but you're not interested in running the lint tools until the implementation is mature.
what do you think about a different tag like this one > ci: lint mock test-unit test-e2e

Reply:
yes, we can do that, agree. but we need to run this always on build to ensure.

also I think in day-to-day coding you will need to run these often otherwise you will have lots of work (fix lint) to do on a mature feature